speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
It wouldn't have anything to do with the matching in the skill since Alexa heard the slot as "the movie ghostbusters".  Very weird that she heard it that way if you have "Ghostbusters" in your MOVIES slot, though.. do you?

It's likely got something to do with the utterances as you've determined, and if I had to guess, it may be a difference in the language parsing between US and UK, since I'm not seeing the same thing.

I have also noticed there is some 'learning' going on that appears to be per-account or per-device, particularly if you bother to utilize the feedback option in the cards.  I think there is some instant adjustment/weighting they make when you say "yes" or "no" there.